vendor/github.com/ONSdigital/go-ns/zebedee/client/client.go

```diff
@@ -244,26 +244,13 @@ func (c *ZebedeeClient) GetPageTitle(ctx context.Context, uri string) (data.Page
 
 func (c *ZebedeeClient) createRequestURL(ctx context.Context, path string) string {
 	var url string
-
-	// Check if collection ID is set in context
 	if ctx.Value(common.CollectionIDHeaderKey) != nil {
 		collectionID, ok := ctx.Value(common.CollectionIDHeaderKey).(string)
 		if !ok {
 			log.ErrorCtx(ctx, errors.New("error casting collection ID cookie to string"), nil)
 		}
 		url = "/data/" + collectionID
 	}
-
 	url = url + path
-
-	// Check if locale code is set in context and add lang query param to url
```

Comment:
We probably still want this so worth re-updating your vendor for the latest version of the client.